package.json

```diff
@@ -70,7 +70,8 @@
     "htmlparser2": "^7.2.0",
     "localforage": "^1.10.0",
     "lodash": "4.17.21",
-    "metro-config": "^0.64.0",
+    "memoize-one": "^6.0.0",
+    "metro-config": "^0.71.3",
```

Comment:
Seems like the `metro-config` upgrade is sneaking in here, despite not being part of https://github.com/Expensify/App/pull/10239 ?